_posts/2020-06-20-snap-minis.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Snap Minis: How we can help you bring your app into SnapChat"
+date: 2020-06-20 11:00:00
+reviewed: 2020-06-20 11:00:00
+categories: []
+author: ABizzinotto
+---
+
+Last week, [Snap](https://www.snap.com) announced a new product during the [Snap Partner Summit 2020](https://www.youtube.com/watch?v=BfvubHha69k), [Snap Minis](https://minis.snapchat.com/). Minis are bite-sized third-party applications that live inside [Snapchat](https://www.snapchat.com/) and can be accessed through Snapchat's Chat interface. They are built on the same principle and technology as the Snap Games and are easy to develop, allowing you to have an app that takes advantage of the existing [Snapchat](https://www.snapchat.com/) application and is available to Snapchat users worldwide.
+
+OmbuLabs recently developed a Snap Mini application for one of our clients. In this article I'll share with you the main advantages of launching a Snap Mini and how we can help you get in on this.
+
+<!--more-->
+
+First of all, why create a Snap Mini? Here are the main advantages of having your own app inside [Snapchat](https://www.snapchat.com/):
+
+## Easy to Develop
+Snap Minis exist inside the main Snapchat application. They are built on HTML5 and take advantage of the existing Snapchat app, which means they work cross-platform and are readily available to all Snapchat users, reducing the scope of the project.
+
+Additionally, Minis work on both Android and iOS devices. You do not need two different applications to cater to different users.
+
+## Easy to Use
+The Minis are lightweight applications easily accessible through the Chat interface of the Snapchat app. Users do not need to install a separate app and switch applications to use your service. They also don't need to wait for a download to finish.
```

Comment:
Here, you have to wait for the download of the mini though when you open it.

And also the same thing on opening the mini through the search and not the chat applies here.

Reply:
@arielj same as above. This is exactly the language Snap used. Users do not need to install applications and wait for downloads. They open the mini and start using it. Same reason as above, I am not deviating from the language they are using.